schemathesis_hooks.py

```
@@ -0,0 +1,42 @@
+
+@schemathesis.hook
+def filter_body(context, body):
+    node = body.definitions[0].selection_set.selections[0]
+    if node.name.value == "createUser":
+        for argument in node.arguments:
+            if argument.name.value == "createUserInput":
+                for field in argument.value.fields:
+                    if field.name.value == "name":
+                        if (field.value.value == "" or field.value.value in users_name):
+                            return False
+                        return True
```

Ruff

Return the negated condition directly

Inline condition

(SIM103)